### PR TITLE
updated Readme

### PR DESCRIPTION
**Pull request by gracyashhh**
Have added some more details about the working of the site

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
-<h2>This is a fully functional ToDo List app made using NodeJs , Express & MongoDB Atlas</h2>
+* This is a fully functional ToDo List app made using 
+    * NodeJs 
+    * Express 
+    * MongoDB
+    * Atlas
```
Comment (alphaCoder-69) [inline README.md:5]: Its a single point ... MongoDB Atlas

Comment (gracyashhh) [reply README.md:5]: oh I mistook that and😅 will change that now

**Comment by gracyashhh** [comment]
Is it fine now?

**Comment by alphaCoder-69** [comment]
yupp..

**Comment by gracyashhh** [comment]
Could you please add the hacktoberfest-accepted label, coz it shows your repo is ineligible without that label on this PR in the official site

**Comment by alphaCoder-69** [comment]
ohh.. sry for that

**Pull request merged**